t/test01.t

```diff
@@ -8,6 +8,15 @@ use Zonemaster::Engine;
 use JSON::PP;
 use File::ShareDir qw[dist_file];
 
+# Use the TARGET environment variable to set the database to use
+# default to SQLite
+my $db_backend = $ENV{TARGET};
+if ( not $db_backend ) {
+    $db_backend = 'SQLite';
+} elsif ( $db_backend !~ /SQLite|MySQL|PostgreSQL/ ) {
+    BAIL_OUT( "Unsupported database backend: $db_backend" );
+}
```

Comment:
Could you add a diagnostic here so it's visible from the TAP output which database engine is beign used?

Reply:
Done in 3896241